Plugins/NativeEngine/CMakeLists.txt

```diff
@@ -26,7 +26,8 @@ set(SOURCES
 
 add_library(NativeEngine ${SOURCES})
 
-target_include_directories(NativeEngine PUBLIC "Include")
+target_include_directories(NativeEngine PUBLIC "Include"
+    PRIVATE "${BIMG_DIR}/3rdparty")
```

Comment:
This seems a little odd. Could we instead either expose STB as a target from bgfx.cmake? Or, if not there, could we add a bespoke target in dependencies/CMakeLists.txt like we do for base-n? I just think it might be best to remove the responsibility of understanding the structure of bimg from NativeEngine.

Reply:
3rdparty directory already added in bgfx.cmake. I don't know why it didn't work 1st time I tried to include the .h. Anyway, it's fixed now.